src/Silverback.Integration/Messaging/Broker/Consumer.cs

```diff
@@ -210,9 +228,23 @@ public async Task TriggerReconnectAsync()
             Task.Run(
                     async () =>
                     {
-                        await WaitUntilConsumingStoppedAsync().ConfigureAwait(false);
-                        await DisconnectAsync().ConfigureAwait(false);
-                        await ConnectAsync().ConfigureAwait(false);
+                        try
+                        {
+                            await WaitUntilConsumingStoppedAsync().ConfigureAwait(false);
+                            await DisconnectAsync().ConfigureAwait(false);
+                            await ConnectAsync().ConfigureAwait(false);
+
+                            _isReconnecting = false;
```

Comment:
Log the successful reconnect?

Reply:
The `ConnectAsync` will log an INFO already. 👍